cobbler/actions/check.py

```diff
@@ -135,7 +135,19 @@ def check_service(self, status, which, notes=""):
         if notes != "":
             notes = " (NOTE: %s)" % notes
         return_code = 0
-        if self.checked_family in ("redhat", "suse"):
+        if utils.is_supervisord():
+            return_code = utils.subprocess_call("supervisorctl status %s > /dev/null 2>/dev/null" % which,
```

Comment:
I would love if we could use the XMLRPC API for this too.

Reply:
what do you mean by this? I'm not familiar with xmlrpc.

Reply:
I am referring to https://github.com/cobbler/cobbler/pull/2841#discussion_r754260340

Reply:
I'll start a new discussion for this.